stable/gocd/values.yaml

```diff
@@ -168,6 +168,7 @@ server:
 agent:
   # agent.replicaCount is the GoCD Agent replicas Count. Specify the number of GoCD agents to run
   replicaCount: 0
+  deployStrategy: {}
```

Comment:
Can you please add a comment above to explain and probably list out of deployment strategy's acceptable values?

Reply:
@varshavaradarajan done!